src/evict/evict_page.c

```diff
@@ -546,15 +546,17 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
     WT_PAGE *page;
     uint32_t flags;
     bool closing, modified;
-    bool snapshot_acquired;
+    bool snapshot_acquired_for_eviction;
```

Comment:
We are in eviction already, so I'm not finding the extra words here additionally helpful.